x/warp/contract.go

```diff
@@ -151,8 +151,8 @@ func getVerifiedWarpBlockHash(accessibleState contract.AccessibleState, caller c
 // UnpackGetVerifiedWarpMessageInput attempts to unpack [input] into the uint32 type argument
 // assumes that [input] does not include selector (omits first 4 func signature bytes)
 func UnpackGetVerifiedWarpMessageInput(input []byte) (uint32, error) {
-	// We don't use strict mode here because it was disabled with the DUpgrade.
-	// Since Warp will be deployed after the DUpgrade, we don't need to use strict mode.
+	// We don't use strict mode here because it was disabled with the Durango.
+	// Since Warp will be deployed after the Durango, we don't need to use strict mode.
```

Comment:
```suggestion
	// We don't use strict mode here because it was disabled with Durango.
	// Since Warp will be deployed after Durango, we don't need to use strict mode.
```